assets/src/web-stories-block/index.js

```diff
@@ -28,9 +28,11 @@ import { registerBlockType } from '@wordpress/blocks';
  * Internal dependencies
  */
 import { name, settings } from './block';
+import { webStoriesBlockSettings } from './block/globals';
+import './block/storiesFilters';
 
-__webpack_public_path__ = global.webStoriesEmbedBlockSettings.publicPath;
+__webpack_public_path__ = webStoriesBlockSettings.publicPath;
 
 registerBlockType(name, settings);
 
-initializeTracking('Embed Block', false);
+initializeTracking('Web Stories List Block', false);
```

Comment:
```suggestion
initializeTracking('Embed Block', false);
```

Don't want to skew our telemetry